Doc/faq/programming.rst

```diff
@@ -549,12 +549,12 @@ desired effect in a number of ways.
 
       args = {'a': 'old-value', 'b': 99}
       func3(args)
-      print(args['a'], args['b'])
+      print(args['a'], args['b'])     # output: new-value 100
 
 5) Or bundle up values in a class instance::
 
       class callByRef:
-          def __init__(self, /, **args):
+          def __init__(self, **args):
```

Comment:
@serhiy-storchaka you added the `/` separator in #13720, but I don’t see why it’s useful (`__init__` being called implicitly with positional `self` param).  Mind if it’s removed?

Reply:
It is needed because you can pass keyword arguments with arbitrary names, and "self" is not an exception.
```python
args = callByRef(self=1)
assert args.self == 1
```
Without `/` the above example fails.

Reply:
@serhiy-storchaka Well but I test in my env, and get `SyntaxError`, did I miss something?
<img width="505" alt="Screen Shot 2020-07-07 at 00 11 49" src="https://user-images.githubusercontent.com/15820530/86615023-75ebea80-bfe6-11ea-93aa-5ce335c81c4c.png">